### PR TITLE
Xenowear Update

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Adds footwear for clawed xenos, as well as a selection of capes for Skrell.
sprites from other server.

image

image

caligae with white socks. you know how to check the icon bot, don't you?

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/loadout_xeno.dm [hunk 1]
```diff
@@ -426,4 +426,34 @@
...
+/datum/gear/shoes/caligae/New()
+	..()
+	var/caligae = list()
+	caligae["no sock"] = /obj/item/clothing/shoes/caligae
+	caligae["black sock"] = /obj/item/clothing/shoes/caligae/black
+	caligae["grey sock"] = /obj/item/clothing/shoes/caligae/grey
+	caligae["white sock"] = /obj/item/clothing/shoes/caligae/white
+	gear_tweaks += new/datum/gear_tweak/path(caligae)
+
+/datum/gear/accessory/capes
```
Comment (Alberyk) [inline code/modules/client/preference_setup/loadout/loadout_xeno.dm]: This has to go in the skrell section.

**Comment by ParadoxSpace** [comment]
touched up some sprites, did what alb said

File: html/changelogs/paradoxspace-romulus.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#################################
+
+# Your name.  
+author: N3X15
```
Comment (skull132) [inline html/changelogs/paradoxspace-romulus.yml]:
```suggestion
author: ParadoxSpace
```

You have no idea how hard it is to not meme this with a horrible name suggestion.

**Comment by ParadoxSpace** [comment]
conflicts and changelog fixed

**Comment by skull132** [comment]
Marked "DO NOT MERGE" until the extended server leadership figure out a proper standard for clothing PRs, due to recent events. Expect an answer by the end of this week.

**Comment by ParadoxSpace** [comment]
can i get an update

File: code/modules/clothing/under/accessories/accessory.dm [hunk 3]
```diff
@@ -336,6 +335,7 @@
 	icon_state = "classicponcho"
 	item_state = "classicponcho"
 	icon_override = 'icons/mob/ties.dmi'
+	var/fire_resist = T0C+100
```
Comment (Alberyk) [inline code/modules/clothing/under/accessories/accessory.dm]: Remove this var, it is useless.

Comment (ParadoxSpace) [reply code/modules/clothing/under/accessories/accessory.dm]: no idea how that got here but ok

**Pull request merged**